apps/aecore/src/aec_blocks.erl

```diff
@@ -111,7 +111,7 @@ new_with_state(LastBlock, Txs, Trees1) ->
     %% We should not have any transactions with invalid signatures for
     %% creation of block candidate, as only txs with validated signatures should land in mempool.
     %% Let's hardcode this expectation for now.
-    {Txs, _} = {aetx_sign:filter_invalid_signatures(Txs), Txs},
+    {Txs, _} = {aetx_sign:filter_invalid_signatures(Txs, Trees1), Txs},
```

Comment:
I see no point in keeping this call as the signatures are checked in the call to `aec_trees:apply_signed_txs/4`. The `aetx_sign:filter_invalid_signatures/1` function is then not needed anymore, which is good since it is no longer correct.